README.rst

```diff
@@ -1,8 +1,8 @@
 django-bitfield
 ---------------
 
-.. image:: https://api.travis-ci.org/disqus/django-bitfield.png?branch=master
-    :target: https://travis-ci.org/disqus/django-bitfield
+.. image:: https://api.travis-ci.org/znotdead/django-bitfield.png?branch=master
```

Comment:
Yeah, because it make total sense to stay on old, unmaintained versions of Django… If they want to use deprecated versions of Django, why don't they use old versions of django-bitfield and let others let the project progress.
I don't have an issue with the project moving github repos but the real problem is they have ownership of the project on PyPI. I have to check if they're is a way to report a project on PyPI and take over the ownership. Otherwise, the only alternative is to rename the project (django-flagfield maybe?)